integration_tests/test_python_plugin.py

```diff
@@ -83,6 +83,16 @@ def test_build_rewrites_shebangs(self):
         self.assertEqual('#!/usr/bin/env python2', python2_shebang)
         self.assertEqual('#!/usr/bin/env python3', python3_shebang)
 
+    def test_build_does_not_keep_pyc_files_in_install(self):
+        project_dir = 'python-entry-point'
```

Comment:
This is a project with both py2 and py3 parts, right?
The name "python-entry-point" does not tell the purpose of the snap in this test. Maybe create a new one? If you rename it, then it will not be clear for the entry point tests.